### PR TITLE
Make Set-GceInstance -RemoveDisk take a Disk object.

### PR DESCRIPTION
**Pull request by ILMTitan**
Adding this attribute allows -RemoveDisk to take a disk object, as well as a string name.

### CODE REVIEW

**Comment by ILMTitan** [comment]
This resolves issue #114

File: Google.PowerShell/Common/GcloudAttributes.cs [hunk 1]
```diff
@@ -117,4 +119,37 @@ public void SetConfigDefault(FieldInfo field, object instance)
             }
         }
     }
+    /// <summary>
+    /// This attribute allows an arry parameter to accept multiple types by replacing a target type with the
```
Comment (chrsmith) [inline Google.PowerShell/Common/GcloudAttributes.cs]: s/arry/array/

File: Google.PowerShell/Common/GcloudAttributes.cs [hunk 2]
```diff
@@ -117,4 +119,37 @@ public void SetConfigDefault(FieldInfo field, object instance)
...
+        public ArrayPropertyTransformAttribute(Type typeToTransform, string property)
+        {
+            typeTransformationAttribute = new PropertyByTypeTransformationAttribute
+            {
+                Property = property,
+                TypeToTransform = typeToTransform
+            };
+        }
+
+        public override object Transform(EngineIntrinsics engineIntrinsics, object inputData)
```
Comment (chrsmith) [inline Google.PowerShell/Common/GcloudAttributes.cs]: This snippet of code is pretty badass. You should have this snippet engraved on your tombstone or a tatoo or something.

**Comment by chrsmith** [comment]
Nice. LGTM

**Pull request merged**